package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "subspace.js",
+  "version": "1.0.0",
```

Really early for 1.0 :slightly_smiling_face: 
```suggestion
  "version": "0.1.0",
```